### PR TITLE
feat(slo-rules-generator): support multiple configurations for single domain

### PR DESCRIPTION
**Pull request by david-vavra**
As it is now, it is not possible to define separate configs for single domain, for example when using multiple SLO versions.

### CODE REVIEW

**Comment by david-vavra** [comment]
I've changed name of the newly added configuration field to `domain`.

File: tools/slo-rules-generator/slo-rules-generator.go [hunk 1]
```diff
@@ -55,39 +54,36 @@ func main() {
...
-				os.Exit(1)
-			}
-			_, err = f.Write(data)
-			if err != nil {
-				fmt.Printf("Error while writing to %s: %v", fname, err)
-				os.Exit(1)
-			}
+	for configName, domainConf := range conf {
+		if errs := domainConf.IsValid(); len(errs) > 0 {
+			fmt.Printf("Error while validation '%s' configuration:\n%v", configName, errs)
```
Comment (rudo-thomas) [inline tools/slo-rules-generator/slo-rules-generator.go:59]:
```suggestion
			fmt.Printf("Error while validating '%s' configuration:\n%v", configName, errs)
```

File: tools/slo-rules-generator/slo-domains.yaml.example [hunk 2]
```diff
@@ -1,4 +1,5 @@
-userportal-reports: # domain-name
+userportal-reports-v6: # config-name
+  domain: userportal-reports
```
Comment (rudo-thomas) [inline tools/slo-rules-generator/slo-domains.yaml.example:2]: The `domain` could default to the config name -- to stay backwards compatible...

**Review by rudo-thomas — APPROVED** [review]

**Comment by david-vavra** [comment]
Pipeline job `build_docker` was failing due to invalid signatures of deb repo. It seems that CircleCI runtime has old images for the version of docker we had been using.

Changed this in [959a019](https://github.com/seznam/slo-exporter/pull/105/commits/959a0195318759393c50032d616567a7b5364995)

https://discuss.circleci.com/t/apt-get-update-no-pubkey/48378/4

File: tools/slo-rules-generator/slo-domains.yaml.example (hunk 2 above)
Comment (david-vavra) [reply tools/slo-rules-generator/slo-domains.yaml.example]: Fixed in 1fcde6ac95f9cb9c1101b86092fb64e53ff70cce

File: tools/slo-rules-generator/slo-rules-generator.go (hunk 1 above)
Comment (david-vavra) [reply tools/slo-rules-generator/slo-rules-generator.go]: Fixed.

**Pull request merged**